extension-manifest-v3/src/utils/engines.js

```diff
@@ -21,6 +21,8 @@ import {
 import { registerDatabase } from '/utils/indexeddb.js';
 
 export const CUSTOM_ENGINE = 'custom-filters';
+export const FIXES_ENGINE = 'fixes';
+export const TRACKERDB_ENGINE = 'trackerdb';
```

Comment:
Not sure if we need to specify trackerDB name as an export. It is used only in `trackerdb.js`. Not here at all.

Reply:
Ok